projects/contracts/src/lib/types.ts

```diff
@@ -154,6 +154,11 @@ export interface Config {
   mediaFolder:   string;
   twitchChannel: string;
   twitchLog?: boolean;
+  twitch: {
+    bot?: boolean;
```

Comment:
I think that setting should be also moved into `twitch.bot` that way if we have another login for the channel broadcaster itself - see #153  - it would have to be saved as well

Also the current `bot` could be `enabled`